src/main/java/de/hysky/skyblocker/skyblock/itemlist/recipebook/SkyblockRecipeBookWidget.java

```diff
@@ -34,8 +35,9 @@ public class SkyblockRecipeBookWidget extends RecipeBookWidget<NoopRecipeScreenH
 	private static final int IMAGE_HEIGHT = RecipeBookWidget.field_32409;
 	//Corresponds to field_32410 in RecipeBookWidget
 	private static final int OFFSET_X_POSITION = 86;
-	//81 is the search field's width, 4 is the space between it and the toggle crafting button, and 26 is the toggle crafting button's width
-	private static final int SEARCH_FIELD_WIDTH = 81 + 4 + 26;
+	// 81 is the search field's width, 4 is the space between it and the toggle crafting button, and 26 is the toggle crafting button's width, which we replace
+	// with the filtering button. (26 - 14) = 12 - 4 = 8
```

Comment:
This confused me a bit lol.
```suggestion
	// with the filtering button. (26 - 14) - 4 = 12 - 4 = 8
```